Update QGIS BugTracker link

agiudiceandrea opened the pull request:
<!---
Include a few sentences describing the overall goals for this Pull Request.
 
A list of issues is at https://github.com/qgis/QGIS-Documentation/issues.
Add "fix #issuenumber" for each issue the PR fixes. The ticket(s) will be closed automatically.
If your PR doesn't fix entirely the ticket, only add the ticket(s) reference preceded by # character.
-->
Goal:
Replace the old Redmine bugtracker link https://issues.qgis.org/projects/qgis/issues with the new bugtracker link on GitHub https://github.com/qgis/QGIS/issues in the "[Help and Support](https://docs.qgis.org/testing/en/docs/user_manual/preamble/help_and_support.html#bugtracker
)" page of the User Guide/Manual for the sentence:

> While the qgis-users mailing list is useful for general 'How do I do XYZ in QGIS?'-type questions, you may wish to notify us about bugs in QGIS. You can submit bug reports using the QGIS bug tracker at https://issues.qgis.org/projects/qgis/issues..


The following sentences should be updated too or removed:

> If you have found a bug and fixed it yourself, you can submit either a Pull Request on the Github QGIS Project (prefered) or a patch also. The lovely redmine ticketsystem at https://issues.qgis.org/projects/qgis/issues has this type as well. Check the Patch supplied checkbox and attach your patch before submitting your bug. One of the developers will review it and apply it to QGIS. Please don't be alarmed if your patch is not applied straight away -- developers may be tied up with other commitments.

Hints on what to do?

<!---
Indicate whether the fix should be backported to previous release.
Replace the space between square brackets by a `x` to make it checked.
-->
- [x] Backport to LTR documentation is required

<!---
Reviewing is a process done by community members, mostly on a volunteer basis.
We try to keep the overhead as small as possible and appreciate if you help us.
Please read carefully and ensure you comply with our writing guidelines at
https://docs.qgis.org/testing/en/docs/documentation_guidelines/index.html.
Feel free to ask in a comment or the (qgis-community-team mailing list)
[https://lists.osgeo.org/mailman/listinfo/qgis-community-team] if you have troubles with any item.
--->

DelazJ commented:
For a first step, I'd just remove anything related to patch supply. I think the most efficient way today is through pull request so let's mention only this way of contributing. Otherwise I think this should have been reworded in the dev guidelines document.

More globally, I think this chapter is a bit redundant with our website, and in the next page "Contributors & authors" we mention the "Get involved" page where all these channels are covered. I'm hence not sure it's worth keeping it... my2cts!

agiudiceandrea commented:
We could reword the whole paragraph:

> BugTracker
> While the qgis-users mailing list is useful for general ‘How do I do XYZ in QGIS?’-type questions, you may wish to notify us about bugs in QGIS. You can submit bug reports using the QGIS bug tracker at https://issues.qgis.org/projects/qgis/issues. When creating a new ticket for a bug, please provide an email address where we can contact you for additional information.
> 
> Please bear in mind that your bug may not always enjoy the priority you might hope for (depending on its severity). Some bugs may require significant developer effort to remedy, and the manpower is not always available for this.
> 
> Feature requests can be submitted as well using the same ticket system as for bugs. Please make sure to select the type Feature.
> 
> If you have found a bug and fixed it yourself, you can submit either a Pull Request on the Github QGIS Project (prefered) or a patch also. The lovely redmine ticketsystem at https://issues.qgis.org/projects/qgis/issues has this type as well. Check the Patch supplied checkbox and attach your patch before submitting your bug. One of the developers will review it and apply it to QGIS. Please don’t be alarmed if your patch is not applied straight away – developers may be tied up with other commitments.
> 
> Note that if you supply a Pull Request, your change would be more likely be merged into the source code!

as:

> BugTracker
> While the qgis-users mailing list is useful for general ‘How do I do XYZ in QGIS?’-type questions, you may wish to notify us about bugs in QGIS. You can submit bug reports using the [QGIS bug tracker](https://issues.qgis.org/projects/qgis/issues).
> 
> Please bear in mind that your bug may not always enjoy the priority you might hope for (depending on its severity). Some bugs may require significant developer effort to remedy, and the manpower is not always available for this.
> 
> Feature requests can be submitted as well using the same ticket system as for bugs. Please make sure to select the type ``Feature request``.
> 
> If you have found a bug and fixed it yourself, you can submit either a Pull Request on the [GitHub QGIS Project](https://github.com/qgis/QGIS).
> 
> Read [Bugs, Features and Issues](https://qgis.org/en/site/getinvolved/development/bugreporting.html#bugs-features-and-issues) for more details.

DelazJ commented:
> We could reword the whole paragraph:
> [...]

👍 
The only concerns are that the "QGIS bug tracker" url is not the right one. And if for issues we use a url with "/issues", we should also pull-requests to their direct page and not the repo main page.
What does :ref:`submit_patch` refer to?

All that said, I am more and more convinced that all these information do not belong to the User manual.

agiudiceandrea commented:
> The only concerns are that the "QGIS bug tracker" url is not the right one.
Yes, it was a oversight.

> What does :ref:`submit_patch` refer to?

A link to https://docs.qgis.org/testing/en/docs/developers_guide/git.html#submitting-patches-and-pull-requests

DelazJ commented:
OK

agiudiceandrea commented:
Hi @havatv, do you think it could be fine now?

havatv commented on docs/user_manual/preamble/help_and_support.rst:
```diff
@@ -93,30 +93,20 @@ BugTracker
-https://issues.qgis.org/projects/qgis/issues. When creating a new
-ticket for a bug, please provide an email address where we can contact
-you for additional information.
+QGIS. You can submit bug reports using the `QGIS bug tracker <https://github.com/qgis/QGIS/issues>`_.
```
```suggestion
QGIS. You can submit bug reports using the
`QGIS bug tracker <https://github.com/qgis/QGIS/issues>`_.
```

havatv commented on docs/user_manual/preamble/help_and_support.rst:
```diff
@@ -93,30 +93,20 @@ BugTracker
+If you have found a bug and fixed it yourself, you can submit a
+Pull Request on the `Github QGIS Project <https://github.com/qgis/QGIS>`_.
+
+Read `Bugs, Features and Issues <https://qgis.org/en/site/getinvolved/development/bugreporting.html#bugs-features-and-issues>`_ and :ref:`submit_patch` for more details.
```
```suggestion
Read
`Bugs, Features and Issues <https://qgis.org/en/site/getinvolved/development/bugreporting.html#bugs-features-and-issues>`_
and :ref:`submit_patch` for more details.
```

havatv commented:
> Hi @havatv, do you think it could be fine now?

Yes. But perhaps @DelazJ would like to have a look before we merge?

DelazJ commented on docs/user_manual/preamble/help_and_support.rst:
```diff
@@ -93,30 +93,20 @@ BugTracker
+as for bugs. Please make sure to select the type ``Feature request``.
+
+If you have found a bug and fixed it yourself, you can submit a
+Pull Request on the `Github QGIS Project <https://github.com/qgis/QGIS>`_.
```
```suggestion
Pull Request on the `Github QGIS Project <https://github.com/qgis/QGIS/pulls>`_.
```

havatv commented:
Thanks, @agiudiceandrea!
